lib/models/grid_items_list.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class GridItemModel {
+  String name;
+  String image;
+  final price;
+  GridItemModel({
+    required this.name,
+    required this.image,
+    required this.price,
+  });
+}
+
+List<GridItemModel> gitems = [
```

Comment:
please rename gitems to something meaningful, so when other people read your code, its easy to understand